dsalgo/Search.py

```diff
@@ -0,0 +1,87 @@
+import sys
+
+class Search:
+
+	def __new__(self, array, algorithm, element):
+
+		'''
+		args:
+	      :array: (list) : a python list
+	      :algorithm: (str): Searching algorithm type
+	                        Values supported are:
+	                        1. linear
+	                        2. binary
+
+	      :element: Element whose index has to be found
+	                if present  
+	    
+	    return:
+	        Index of the element if found else returns -1
+		'''
+
+
+		self.array = array
+		self.algorithm = algorithm
+		self.element = element
+
+		if self.algorithm == 'linear':
+			return linear_search(self.array, self.element)
+
+		elif self.algorithm == 'binary':
+			return binary_search(self.array, 0, len(self.array)-1, self.element)
+
+		else:
+			stderr.write("Error: unsupported searching algorithm passed!")
```

Comment:
if you have tested your code properly then you must understand the error in this line.
Don't do contributions for pull request do contributions for collaboration.

Reply:
I have missed the "sys" import when I did the commit. I'll correct it. 
I am doing it for collaboration and that's the reason I am doing it before October. :)